src/types/mod.rs

```diff
@@ -1,3 +1,7 @@
+//
```

Comment:
I think nowadays it's more idiomatic to have a `type.rs` file at the same directory level and the `types` directory and then only include the specific submodules inside the `types` directory.